### DISCUSSION TITLE
Windows app doesn't open on first note in list

### DISCUSSION THREAD

**Discussion by thetrashcanhero** [standardnotes/forum#3130] [General]
**Describe the bug**
On my Windows 11 app, it doesn't open to the first app on the list when the option is selected.

**To Reproduce**
1. Open App on Windows 11
2. Click on "Open Preferences"
3. Under "When opening the app, show..." select "The first note in the list"
4. Close app
5. Reopen app

**Expected behavior**
To see the first note on the list of my notes

**Actual behavior**
It does not open on the first note

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: n/a, it's on the native SN app
 - Version: 3.144.3

**Comment by effieeee** [comment]
Hi @thetrashcanhero, It's been a while since you opened this, so I apologize that no one was able to respond. Are you perhaps still experiencing this on v3.160.19 of the desktop app?

**Reply by thetrashcanhero** [reply]
Yes still experiencing it on that version. When "the first note in the list" is checked, it opens the latest note I created.

**Reply by effieeee** [reply]
I see, upon testing it as well, I was able to reproduce it on the latest desktop app and web app, so I'll also consult the team about this so they can take a closer look 🙏
